config/locales/alchemy.it.yml

```diff
@@ -0,0 +1,924 @@
+    external_link_notice_2: "Per fare riferimento ad un indirizzo interno al tuo sito, comincia il link con /."
+    female: "Femmina"
+    file: "File"
+    file_rename_error: "Errore nel modificare il nome del file."
+    files: "File"
+    from_at: "da %{by} a %{at}"
+    height: "Altezza"
+    help:
+      shortcuts:
+        "Open help window": "Apri finestra di aiuto"
+        "Close current dialog": "Chiudi finestra attiva"
+        "Open logout dialog": "Apri finestra di logout"
+        "Focus search field": "Attiva il campo di ricerca"
+        "Create a new record": "Crea un nuovo record"
+        "Show page infos": "Mostra informazioni pagina"
+        "Create new element": "Crea nuovo elemento"
+        "Edit page properties": "Modifica proprietà pagina"
+        "Reload the preview": "Ricarica la preview"
+        "Leave the page": "Lascia la pagina"
+        "Open upload form": "Apri form di upload"
+        "Select all pictures": "Seleziona tutte le immagini"
+    hide_element_content: "Nascondi il contenuto di questi elementi."
+    dashboard: "Dashboard"
+    image_alt_tag: "Alt-tag"
+    image_caption: "Caption"
+    image_name: "Name"
+    image_title: "Title-tag"
+    images: "Immagini"
+    internal_link_headline: "Scegli una pagina a cui linkare."
+    internal_link_page_elements_explanation: "In aggiunta puoi cliccare appena sotto una pagina su 'Mostra Elementi' per linkare ad un anchor di un elemento da quella pagina."
+    internal_link_page_anchors_explanation: "Alternativamente puoi linkare ad un anchor della pagina stessa."
+    "item copied to clipboard": "%{name} copiato nella clipboard"
+    "item moved to clipboard": "%{name} spostato alla clipboard"
+    "item removed from clipboard": "%{name} eliminato dalla clipboard"
+    javascript_disabled_headline: "Javascript disabilitato!"
+    javascript_disabled_text: "Javascript è richiesto per far funzionare Alchemy al meglio. Prego abilitalo nelle configurazioni del tuo browser."
+    language_code_placeholder: 'i.e. it'
+    language_does_not_exist: "Questo albero della lingua non esiste"
+    language_pages_copied: "Albero della lingua copiato con successo."
+    last_upload_only: "Solo ultimo upload"
+    left: "sinistra"
+    legacy_url_info_text: "Un link è un redirect da una vecchia URL alla nuova URL di questa pagina. Questo redirect viene gestito tramite uno <a href='https://support.google.com/webmasters/answer/93633' target='_blank'>status code 301</a>."
+    link_image: "Crea collegamento a questa immagine."
+    link_overlay_tab_label:
+      contactform: "Form dei contatti"
+      external: "Esterno"
+      file: "File"
+      internal: "Interno"
+    link_title: "Titolo link"
+    login: "login"
+    logout: "logout"
+    mail_to: "Destinatario"
+    main_content: "Contanuto principale"
+    male: "Maschio"
+    me: "Me"
+    medium_thumbnails: "Thumbnails di meda dimensione"
```

Comment:
I say we remove the 's' (which is wrong of course) but keep the English term.

It seems to me it's widely accepted on the Italian web slang: https://it.wikipedia.org/wiki/Thumbnail